hack/test-integration.sh

```diff
@@ -50,5 +54,8 @@ runTests() {
 # Run cleanup to stop etcd on interrupt or other kill signal.
 trap cleanup EXIT
 
-runTests "v1beta1"
-runTests "v1beta3"
+# Convert the CSV to an array of API versions to test
+IFS=',' read -a apiVersions <<< ${KUBE_TEST_API_VERSIONS}
+for apiVersion in "${apiVersions[@]}"; do
+  runTests $apiVersion "${@}"
```

Comment:
you didn't previously pass "${@}" to runTests - was this an intentional addition? (It seems like it doesn't really have much effect in any case)